src/message_dao_test.py

```diff
@@ -0,0 +1,79 @@
+import unittest
+import subprocess
+import zlib
+
+from unittest import mock
+from google.cloud.sql.connector import Connector
+
+import message
+import message_dao
+import archive_converter
+from test_helpers import test_data_path
+
+class MessageDaoTest(unittest.TestCase):
+
+    def setUp(self):
+        self.addCleanup(mock.patch.stopall)
+        self.mock_connect = mock.patch.object(Connector, 'connect').start()
+        mock_connection = self.mock_connect.return_value
+        self.mock_commit = mock_connection.commit
+        self.mock_cursor = mock_connection.cursor.return_value.__enter__.return_value
+        self.mock_execute = self.mock_cursor.execute
+        # Check initialization of DAO
+        self.dao = message_dao.MessageDao('FAKE_GIT_PATH')
+        self.mock_connect.assert_called_once()
+        self.mock_commit.assert_called_once()
+        self.assertEqual(3, self.mock_execute.call_count)
+        self.mock_connect.reset_mock()
```

Comment:
very nit: we probably don't need to reset the connect mock since we're overriding side effect to raise an error

Reply:
Yeah, it's unnecessary so it's been removed to improve readability.